### DISCUSSION TITLE
How to read fine_tune's DPA-2 model for distillation in dpgen

### DISCUSSION THREAD

**Discussion by 96XinHe** [deepmodeling/dpgen#1675] [Q&A]
I have pre-trained and fine-tuned the DPA-2 model in deepmd-kit (3.0.0b4). 
When I try to implement the distillation process using dpgen (0.12.2.dev1), the following problem occurs:

I want to use the `.pt` model from the fine-tuning as the initial model and have specified the following parameters in the run parameter.json:
```
    “numb_models": 4,
    “train_backend": ‘pytorch’,
    “training_init_model": true,
    “training_iter0_model_path“:[”./work_path/training_iter0_model_path/1” ,
“./work_path/training_iter0_model_path/2” ,
“./work_path/training_iter0_model_path/3”,
“./work_path/training_iter0_model_path/4”
                                ],
```
The following error message appears:
**FileNotFoundError: cannot find upload file ./iter.000000/00.train/000/old/model.ckpt.meta**

Does this mean that only TensorFlow models can be used as initial models? Or am I missing some parameter?

**Comment by njzjz** [comment]
Do you use the devel branch?
